backend/proxy_server/main.go

```diff
@@ -27,7 +27,7 @@ func main() {
 	setAPIGroup(router)
 
 	// Start and run the server
-	if err := router.Run(":80"); err != nil {
+	if err := router.Run(":4000"); err != nil {
```

Comment:
Since the exposed port has been changed. The declaration in yaml should also be changed

https://github.com/openyurtio/yurt-dashboard/blob/9ab2df215adbbc0240075bfa6c259ee3998b1bdf/charts/yurt-dashboard/templates/deployment.yaml#L24